edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_courses.scss

```diff
@@ -1,604 +1,767 @@
-            @include omega(3n);
-          }
-
-          @include media($bp-huge) {
-            @include span-columns(3); // 3 of 12
-            @include omega(4n);
-          }
-        }
-      }
-    }
-  }
+.courses-container .courses .course:hover .cover-image::before, .courses-container .courses .course:focus .cover-image::before {
+ opacity:0;
+}
+
+.btn-learnmore {
+  border-style: solid;
+  border-radius: 0.1875rem;
+  border-width: 1px;
+  box-shadow: none;
+  padding: 0.625rem 1.25rem;
+  font-size: 16px;
+  font-weight: normal;
+  text-shadow: none;
+  color: $inverse;
+  background-color: $primary;
+  margin: 10px 15px 20px 15px;
+  text-align: center;
+}
+
+.btn-learnmore:hover {
+  color: $primary;
+  background-color: $inverse;
 }
 
-// +Hero - Home Header
-// ====================
+
+.courses-container .courses .course .course-info h2 {
+  margin-bottom: 5px;
+}
+
+.courses-container .courses .course .course-info .course-date {
+  margin-bottom: 5px;
+}
+
+.courses-container .courses .course .course-info .course-title {
+//  margin: 10px 0 15px 0;
+  overflow: hidden;
+  -o-text-overflow: ellipsis;
+  text-overflow: ellipsis;
+  display: -webkit-box !important;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
+  line-height: 1.3;
+  text-align: left;
+  height: 45px;
+  padding-top: 5px;
+}
+
+
 .find-courses,
 .university-profile {
-  header.search {
-    background: $gray-l5;
-    background-size: cover;
-    background-image: $homepage-bg-image;
-    background-position: center top !important;
-    border-bottom: 1px solid $border-color-3;
-    box-shadow: inset 0 -1px 8px 0 rgba(0, 0, 0, 0.2), inset 0 1px 12px 0 rgba(0, 0, 0, 0.3);
-    height: 430px;
-    margin-top: $header_image_margin;
-    width: 100%;
+  background: $background-color-courses;
+}
 
-    .inner-wrapper {
-      height: 120px;
-      margin: 0 auto;
-      max-width: map-get($container-max-widths, xl);
-      padding-top: ($baseline*10);
-      position: relative;
+.courses-container .courses .course
+.course-image .cover-image .learn-more {
+  background: $primary;
+}
+
+.courses-container .courses .course .course-info .course-organization,
+.courses-container .courses .course .course-info .course-code,
+.courses-container .courses .course .course-info .course-date {
+  color: $secondary;
+}
+
+div.content-wrapper.main-container {
+  min-height: 442px;
+}
+
+//about
+.course-info header.course-profile .intro-inner-wrapper
+section.intro div.main-cta a strong {
+  background: $primary;
+  box-shadow: none;
+  border: 0;
+  color: $inverse;
+}
+
+.course-info header.course-profile .intro-inner-wrapper .intro .main-cta a.register, .course-info .button-enroll-bottom a.register, .course-info header.course-profile .intro-inner-wrapper .intro .main-cta a.add-to-cart  {
+  border: none;
+  background-image: none;
+  text-decoration: none;
+  text-shadow: none;
+  background-color: $primary;
+  color: #fff;
```

Comment:
Is there already a variable for this value? @luismorenolopera If so, please replace it as necessary.

Reply:
ok, done